R/EstimationDataFromComparativeEffectivenessModel.R

```diff
@@ -0,0 +1,114 @@
+# @file EstimationDataFromComparativeEffectivenessModel.R
+#
+# Copyright 2020 Observational Health Data Sciences and Informatics
+#
+# This file is part of SkeletonComparativeEffectStudy
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+
+
+#' gets a table with data from preference score distribution.
+#'
+#' @details
+#' gets preference score data for target and comparator in data frame format from 
+#' OHDSI comparative effectiveness data model. The output maybe used to create plots or tables
+#' that compares the preference score distribution between target and comparators.
+#' 
+#' @param connectionDetails    (optional) An object of type \code{connectionDetails} as created using the
```

Comment:
connectionDetails is not an argument of this function. Again, R check would already have flagged this.